server/types/models/video/video.ts

```diff
@@ -219,3 +219,8 @@ export type MVideoFormattableDetails =
   Use<'VideoStreamingPlaylists', MStreamingPlaylistRedundanciesOpt[]> &
   Use<'VideoFiles', MVideoFileRedundanciesOpt[]> &
   PickWithOpt<VideoModel, 'Trackers', MTrackerUrl[]>
+
+export enum VideoStorageType {
```

Comment:
I don't think this is the best location for this enum but I wasn't sure where else to put it

Reply:
Put it in `shared/models/videos/video-storage.type.ts`